index.bs

```diff
@@ -10754,11 +10754,26 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				1. If <var>node</var> is a {{DelayNode}} that is part of a cycle, add it
 					to <var>cycle breakers</var> and remove it from <var>nodes</var>.
 
-			5. If <var>nodes</var> contains cycles, [=mute=] all the
+			5. For each {{DelayNode}} <var>delay</var> in <var>cycle breakers</var>:
+
+				1. Let <var>delayWriter</var> and <var>delayReader</var> be two
+					placeholder {{AudioNode}} that represent the two ends of the delay
+					line. <var>delayWriter</var> has the same input {{AudioNode}} as
+					<var>delay</var>. <var>delayReader</var> is connected to the same
```

Comment:
Also the behavior of `delayWriter` and `delayReader` is not described in the text. Just like the rest of AudioNode types, we have to define how they behave when it is processed.

Reply:
> But also doesn't it mean that we have to remove the actual `delay` from `nodes` after adding two proxies?

This is handled in the loop just before.

Reply:
> Also the behavior of `delayWriter` and `delayReader` is not described in the text. Just like the rest of AudioNode types, we have to define how they behave when it is processed.

I'll come up with something. Incidentally, I think it will help with reconciling channel count and tail-time.

Reply:
> Perhaps `delayWriterProxy` and `delayReaderProxy`?

Let's not use proxy in web specs, it's already complicated enough to not reuse terms that have an historical meaning on the platform.

Reply:
I had no idea the `proxy` was a banned term. :)

Reply:
I don't think it is, I just find it confusing in this context.

Reply:
Also strictly speaking, a proxy wraps something and is not part of the object. Here, the `DelayWriter`s and `DelayReader`s are two interfaces allowing the use an object with specific roles.